crates/namada/src/ledger/mod.rs

```diff
@@ -184,25 +185,22 @@ mod test {
         /// Initialize a test client for the given root RPC router
         pub fn new(rpc: RPC) -> Self {
             // Initialize the `TestClient`
-            let mut wl_storage = TestWlStorage::default();
+            let mut state = TestState::default();
 
             // Initialize mock gas limit
             let max_block_gas_key =
                 namada_parameters::storage::get_max_block_gas_key();
-            wl_storage
-                .storage
-                .write(&max_block_gas_key, namada_core::encode(&20_000_000_u64))
-                .expect(
-                    "Max block gas parameter must be initialized in storage",
-                );
+            state.write(&max_block_gas_key, 20_000_000_u64).expect(
```

Comment:
Shouldn't this be a `db_write`? Probably it doesn't really affect the tests though but I saw that previously we were writing directly t storage while now it looks like we are writing to the block write log

Reply:
yeah, the storage read will still find it, but it really should go directly to DB

Reply:
c3c29e5ec22d3a1ae6e0f554b7115297fb25c532